server/apis/v1/handler.go

```diff
@@ -96,6 +96,32 @@ func (h *handler) Callback(c *gin.Context) {
 	h.dexpoc.handleCallback(c)
 }
 
+// Logout is used to remove auth cookie ending a users session.
+func (h *handler) Logout(c *gin.Context) {
+	cookie := &http.Cookie{
+		Name:     "user-identity-token",
+		Value:    "",
+		Path:     "/",
+		Expires:  time.Unix(0, 0),
+		HttpOnly: true,
+	}
+	http.SetCookie(c.Writer, cookie)
```

Comment:
Can we just do `c.SetCookie`? [reference](https://medium.com/@gopal96685/handling-cookies-with-gin-framework-in-go-f119358c9cf3#:~:text=Deleting%20a%20Cookie%20in%20Gin&text=In%20this%20example%2C%20we%20are%20deleting%20the%20%E2%80%9Cuser%E2%80%9D%20cookie,to%20delete%20the%20cookie%20immediately.)